deployment.rst

```diff
@@ -235,6 +235,11 @@ kernel and return your project's root directory::
         }
     }
 
+Out of memory when accessing /web/app.php
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some UNIX systems make use of SELinux, specially Red Hat distributions, like CentOS. By default SELinux is set to enforcing as it should be. If you deploy with app_dev.php you will see some weird permission errors even if you set 777 to `var` directory inside your project's folder. To resolve this, either set SELinux configuration correctly or disable it.
```

Comment:
`either set SELinux configuration correctly or disable it`.

I don't see any value in this if you say to someone `configure it correctly` you aren't really helping them. Also,  SELinux is a security feature, I don't think we should recommend people to disable it.

Reply:
You're, I should have written SELinux commands then or fully omit this phrase. I'll remove that.